server/routes/additional-learning-needs-screener/create/check-your-answers/checkYourAnswersController.test.ts

```diff
@@ -0,0 +1,129 @@
+import type { ReferenceDataItemDto } from 'dto'
+import { mapSelectedTypesToCategories } from './checkYourAnswersController'
+import { asArray } from '../../../../utils/utils'
+
+describe('checkYourAnswersController', () => {
+  // TODO write controller unit tests in next PR
+})
+
+describe('mapSelectedTypesToCategories', () => {
```

Comment:
These tests are important ... there is a function in the controller (next file down) that takes the array of selected challenges or strengths, and the corresponding reference data, and returns an object that contains the selections grouped by their category (I guess it's kind of like a subset of the ref data, containing just the selections, and no empty categories)

Anyway, the function currently exists in the CheckYourAnswers controller because thats where it is currently used, but I have a feeling we might need this grouping elsewhere in the future (perhaps on the profile page/tab that displays the screener), so this is why it is an exported function in its own right, and we might choose to move it out of this controller at some point